bindings/rust/aws-lc-fips-sys-template/deps/aws-lc/include/rust_wrapper.h

```diff
@@ -100,6 +100,7 @@ int ERR_GET_FUNC_RUST(uint32_t packed_error);
 #include "openssl/sha.h"
 #include "openssl/siphash.h"
 #include "openssl/span.h"
+#include "openssl/ssl.h"
```

Comment:
If we add `#include "openssl/ssl3.h"` here, do we still need to create `rust_ssl_wrapper.h`?

Reply:
I don't think we want `ssl.h` included here -- it will cause bindings for the ssl library to be generated when we're not building the ssl library. 

I'm curious why this line was added.